finout/manifest.json

```
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fc1d0074-ccd3-4513-a5c3-3d8ee99b1e49",
+  "app_id": "finout",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Offering::UI Extension"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "The modern cost observability platform for FinOps",
+    "title": "Finout",
+    "media": [
+      {
+        "caption": "Finout Cost Observability",
+        "image_url": "images/DD-banner.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Kubernetes level cost visibility",
+        "image_url": "images/DD-fig2.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Control exactly what you want to see",
+        "image_url": "images/DD-configuration.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Customize your own visualization",
+        "image_url": "images/DD-fig1.png",
+        "media_type": "image"
+      }
+  ]
+  },
+  "author": {
+    "support_email": "support@finout.io"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "Finout",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "finout.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      }
+    }
+  }
```

```suggestion
  "assets": {}
```
This section can be deleted.

If you choose to add a dashboard, then it can be added to the assets section.